lib/srv/desktop/windows_server.go

```diff
@@ -519,6 +540,10 @@ func (s *WindowsService) startStaticHostHeartbeats() error {
 // Close instructs the server to stop accepting new connections and abort all
 // established ones. Close does not wait for the connections to be finished.
 func (s *WindowsService) Close() error {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	s.ldapCertRenew.Stop()
```

Comment:
Is there any chance `s.ldapCertRenew` will be `nil` here? Can I `Close` before the first successful call to `initializeLDAP`?

Reply:
Good catch, it is possible. Unlikely, but possible. Fixed.